aframe/tasks/data/timeslide_waveforms/timeslide_waveforms.py

```diff
@@ -47,7 +47,7 @@ class DeployTimeslideWaveforms(
     StaticMemoryWorkflow,
 ):
     def workflow_requires(self):
-        reqs = super().workflow_requires()
+        reqs = {}
```

Comment:
Can you explain why you made this change?

Reply:
Mostly b/c we don't do need anything from the parent class and I thought it was more confusing